.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 2
```

Comment:
Done—I've left the commit in this PR, if you merge that one it'll disappear from this one (or I can rebase it out).